rootfs/etc/cont-init.d/08-svc-snmptrapd.sh

```diff
@@ -2,6 +2,15 @@
 
 SIDECAR_SNNMPTRAPD=${SIDECAR_SNNMPTRAPD:-0}
 LIBRENMS_SNMP_COMMUNITY=${LIBRENMS_SNMP_COMMUNITY:-librenmsdocker}
+SNMP_ACTION=${SNMP_ACTION:-log,execute,net}
+SNMPV3_USER=${SNMPV3_USER:-librenms_user}
+SNMPV3_AUTH=${SNMPV3_AUTH:-auth_pass}
+SNMPV3_PRIV=${SNMPV3_PRIV:-priv_pass}
+SNMPV3_AUTH_PROTO=${SNMPV3_AUTH_PROTO:-SHA}
+SNMPV3_PRIV_PROTO=${SNMPV3_PRIV_PROTO:-AES}
+SNMPV3_SECURITY_LEVEL=${SNMPV3_SECURITY_LEVEL:-priv}
+SNMP_ENGINEID=${SNMP_ENGINEID:-1234567890}
+DISABLE_AUTHORIZATION=${DISABLE_AUTHORIZATION:-yes}
```

Comment:
Would be better to prefix with `SNMP_` to avoid confusion with others env vars:

```suggestion
SNMP_DISABLE_AUTHORIZATION=${SNMP_DISABLE_AUTHORIZATION:-yes}
```

Reply:
This suggestion has been applied